2.6/README.md

```diff
@@ -0,0 +1,145 @@
+Ruby 2.6 container image
+=================
+This container image includes Ruby 2.6 as a [S2I](https://github.com/openshift/source-to-image) base image for your Ruby 2.6 applications.
+Users can choose between RHEL, CentOS and Fedora based builder images.
+The RHEL images are available in the [Red Hat Container Catalog](https://access.redhat.com/containers/),
+the CentOS images are available on [Docker Hub](https://hub.docker.com/r/centos/),
+and the Fedora images are available in [Fedora Registry](https://registry.fedoraproject.org/).
+The resulting image can be run using [podman](https://github.com/containers/libpod).
+
+Note: while the examples in this README are calling `podman`, you can replace any such calls by `docker` with the same arguments
+
+Description
+-----------
+
+Ruby 2.6 available as container is a base platform for
+building and running various Ruby 2.6 applications and frameworks.
+Ruby is the interpreted scripting language for quick and easy object-oriented programming.
+It has many features to process text files and to do system management tasks (as in Perl).
+It is simple, straight-forward, and extensible.
+
+This container image includes an npm utility, so users can use it to install JavaScript
+modules for their web applications. There is no guarantee for any specific npm or nodejs
+version, that is included in the image; those versions can be changed anytime and
+the nodejs itself is included just to make the npm work.
+
+Usage
+---------------------
+For this, we will assume that you are using the `rhscl/ruby-26-rhel7 image`, available via `ruby:2.6` imagestream tag in Openshift.
+Building a simple [ruby-sample-app](https://github.com/sclorg/s2i-ruby-container/tree/master/2.6/test/puma-test-app) application
+in Openshift can be achieved with the following step:
+
+    ```
+    oc new-app ruby:2.6~https://github.com/sclorg/s2i-ruby-container.git --context-dir=2.6/test/puma-test-app/
+    ```
+
+The same application can also be built using the standalone [S2I](https://github.com/openshift/source-to-image) application on systems that have it available:
+
+    ```
+    $ s2i build https://github.com/sclorg/s2i-ruby-container.git --context-dir=2.6/test/puma-test-app/ rhscl/ruby-25-rhel7 ruby-sample-app
```

Comment:
ruby-25-rhel7 looks like another string that should have changed, on this line.  I think the first reviewer was suggesting this PR should start with a clean copy of the 2.5 support forward into `2.6/` without any changes, then layer some commits on top so it's clear to a reviewer what has changed since 2.5